### PR TITLE
[WIP] #432 committee protocol invited section is confusing

### LINKED ISSUE #432: committee protocol invited section is confusing
* goto: https://oknesset.org/committee/meeting/10034/
* look at the "מוזמנים" section

### expected
* person name and title should be on same line

### actual
* person name is separated by line breaks and dash
* it is not clear which title belongs to which person

## notes
* I think it should be fixed when scraping the data, but it might be possible to fix on display as well.. whatever is easier

### PR DESCRIPTION
**Pull request by wasymur**
fixes #432

### CODE REVIEW

**Comment by wasymur** [comment]
1) these 3 functions:
* get_committee_protocol_text()
* handle_doc_protocol()
* handle_rtf_protocol()
were moved to a separate model (committee_protocol.py)

2) add blank line after table rows when parsing committee protocol 

3) remove unnecessary "-" lines between the member name and its summary  , in the create_protocol_parts() level

-------------------

should triple test before merge.

File: simple/management/committee_protocol.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        if flag:
+            logger.error("can't open url %s. tried %d times" % (self._url, count))
+
+        if self._url.find(".rtf") >= 0:
+            return self.handle_rtf_protocol()
+        if self._url.find(".doc") >= 0:
+            return self.handle_doc_protocol()
+
+    def handle_doc_protocol(self):
+        directory = os.path.join(DATA_ROOT, 'comm_p')
```
Comment (OriHoch) [inline simple/management/committee_protocol.py]: this line can be removed

File: simple/management/committee_protocol.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+logger = logging.getLogger("open-knesset.syncdata")
+
+DATA_ROOT = getattr(settings, 'DATA_ROOT',
+                    os.path.join(settings.PROJECT_ROOT, os.path.pardir, os.path.pardir, 'data'))
+
+ENCODING = 'utf8'
+
+
+class CommitteProtocolImporter:
```
Comment (OriHoch) [inline simple/management/committee_protocol.py]: if we are already doing refactoring, then this class should be in committees app

**Comment by OriHoch** [comment]
thanks, good work

please include some notes about what you did, specifically - if this change will be for all exisiting protocols? only from now on? what needs to be done to fix old protocols?

**Comment by OriHoch** [comment]
@wasymur - are you going to continue working on it or should I reassign to someone else?

**Comment by OriHoch** [comment]
I'll work on it

**Comment by OriHoch** [comment]
will be handled in #490

**Pull request closed without merging**